src/OpenTelemetry.Instrumentation.Process/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+## 0.5.0-beta.7
+
+Released 2024-Sep-24
```

Comment:
unless you are planning to request a release right away (which is totally valid!), there is no need to put version number/date.

Reply:
Even if you are planning to request a release right away, this might mess up the automation 🤣 Just add your note under "## Unreleased" and the rest will be taken care of for you automatically.